operator/src/test/java/oracle/kubernetes/operator/helpers/KubernetesUtilsTest.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.helpers;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.*;
```

Comment:
generally we do not allow wildcard imports, though this particular one is probably ok